src/FocusAPI/Methods/Social/AcceptFriendRequest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using FocusAPI.Data;
+using FocusAPI.Models;
+using FocusCore.Commands.Social;
+using FocusCore.Commands.User;
+using FocusCore.Models;
+using FocusCore.Queries.Shop;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace FocusApi.Methods.Social;
+public class AcceptFriendRequest
+{
+    public class Handler : IRequestHandler<AcceptFriendRequestCommand, Unit>
+    {
+        FocusContext _context;
+        ILogger<Handler> _logger;
+        public Handler(FocusContext context, ILogger<Handler> logger)
+        {
+            _context = context;
+            _logger = logger;
+        }
+
+        public async Task<Unit> Handle(AcceptFriendRequestCommand command, CancellationToken cancellationToken)
+        {
+            try
+            {
+                var friendship = await _context.Friends
+                    .Include(f => f.User)
+                    .Include(f => f.Friend)
```

Comment:
Since the user and friend properties aren't used outside of the query itself, including them might slow down the query without providing any use. Could we remove these include statements for efficiency?